scapy/utils.py

```diff
@@ -1649,10 +1649,10 @@ class RawPcapNgReader(RawPcapReader):
                                             ["linktype", "tsresol",
                                              "tshigh", "tslow", "wirelen",
                                              "comment", "ifname", "direction",
-                                             "process_information"])
+                                             "process_information", "comments", ])
 
-    def __init__(self, filename, fdesc=None, magic=None):  # type: ignore
-        # type: (str, IO[bytes], bytes) -> None
+    def __init__(self, filename, fdesc=None, magic=None, comments=None):  # type: ignore
```

Comment:
This looks weird. I don't understand why you would be passing comments to the reader?